packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -72,34 +89,34 @@ def add_networkd_config(src):
         shutil.copyfile(src, dst)
 
     # Restart networkd only if it's active
-    result = subprocess.run(['systemctl', 'is-active', networkd],
-                            stdout=subprocess.PIPE)
-    if result.returncode != 0:
-        result.returncode = 0
-        return result
+    active_check_ret = subprocess.run(['systemctl', 'is-active', networkd],
```

Comment:
do we really need this change?

Reply:
Will these bring error?

Reply:
What I really changed is the return value variable, and the explanation can be found in the above discussion.
> Remember any code that you touch has the potential of introducing a regression
Yeah, we should be as cautious as possible, and you know what, it's just reusing the same variable `result` makes this code much harder to maintain to even let you change a variable.

Reply:
@GoelDeepak , what do you think about this?

Reply:
I think what Deepak meant is that `result` would do the job here too. I am approving the PR, but please consider using just `result` instead of `*_ret`. Thanks.

Reply:
Then I am answering the right question.